city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]/h3'
+        )
+        for title in link_list:
+            full_str = title.xpath(".//text()").extract()[0]
+            date = full_str.split("-")[0].split("(")[0]
+
+            date_obj = datetime.strptime(date.strip(" "), '%B %d, %Y')
+            new_date_obj = datetime(date_obj.year, date_obj.month, date_obj.day, 13, 00)
+            for one_day_links in title.xpath('.//following-sibling::ul[1]/li'):
+                href = one_day_links.xpath('.//@href').get()
+                name = one_day_links.xpath('.//text()').extract()
+                if href is None:
```

Comment:
If there isn't an `href` attribute we should `continue` because we don't want to scrape empty links